app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_components = command.split()
+    source_name = command_components[1]
+    destination_name = command_components[2]
+    is_valid_command = command_components[0] == "mv"
+    is_valid_count_of_components = len(command_components) == 3
+
+    if is_valid_command and is_valid_count_of_components:
+        destination_folders = destination_name.split("/")[:-1]
+        new_file_name = destination_name.split("/")[-1]
+        new_path_to_create = ""
+
+        for dir_name in destination_folders:
+            new_path_to_create = os.path.join(new_path_to_create, dir_name)
+            if not os.path.exists(new_path_to_create):
+                os.mkdir(new_path_to_create)
```

Comment:
nice implementation! Also you can read about `makedirs` from module `os`, for your future projects! (no need to rework, that informational comment.